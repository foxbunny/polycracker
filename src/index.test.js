import * as fpjs from './index'

describe('polycracker', () => {
  test('has the right exports', () => {
    expect(Object.keys(fpjs)).toMatchSnapshot()
  })
})
