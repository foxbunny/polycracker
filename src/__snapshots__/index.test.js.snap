// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`polycracker has the right exports 1`] = `
Array [
  "Any",
  "createAbstractFn",
  "id",
  "withValue",
  "acc",
  "mapAcc",
  "uncurry",
  "cond",
  "maybe",
  "maybeType",
  "maybeVal",
  "maybePass",
  "either",
  "eitherType",
  "eitherPass",
  "eitherVal",
  "match",
  "opt",
  "Type",
  "createBaseType",
  "createType",
  "typeOf",
  "isType",
  "isPrimitive",
  "notNull",
  "map",
  "flatMap",
  "ap",
  "add",
  "sub",
  "mult",
  "div",
  "first",
  "last",
  "slice",
  "tail",
  "startsWith",
  "endsWith",
  "includes",
  "includedIn",
  "filter",
  "len",
  "reverse",
  "empty",
  "zip",
  "every",
  "some",
  "sort",
  "append",
  "appendTo",
  "repeat",
  "weight",
  "lt",
  "lte",
  "gt",
  "gte",
  "eq",
  "similarTo",
  "order",
  "and",
  "or",
  "not",
  "collectInto",
  "iterable",
  "cycle",
  "Maybe",
  "Some",
  "Nothing",
  "Either",
  "Left",
  "Right",
]
`;
